pandas/tests/dtypes/test_common.py

```diff
@@ -392,6 +392,48 @@ def test_is_not_unsigned_integer_dtype(dtype):
     assert not com.is_unsigned_integer_dtype(dtype)
 
 
+@pytest.mark.parametrize(
+    "dtype1,dtype2",
+    [
+        ("int_", "int_"),
+        ("int8", "int8"),
+        ("Int8", "Int8"),
+        ("int16", "int16"),
+        ("Int16", "Int16"),
+        ("int32", "int32"),
+        ("Int32", "Int32"),
+        ("int64", "int64"),
+        ("Int64", "Int64"),
+        ("uint8", "uint8"),
+        ("uint16", "uint16"),
+        ("uint32", "uint32"),
+        ("uint64", "uint64"),
+        ("float16", "float16"),
+        ("float32", "float32"),
+        ("Float32", "Float32"),
+        ("float64", "float64"),
+        ("Float64", "Float64"),
+    ],
+)
+def test_all_type_comparison_with_true_result(dtype1, dtype2):
+    assert pandas_dtype(dtype1) == dtype2
+
+
+@pytest.mark.parametrize(
+    "dtype1,dtype2",
+    [
+        ("Int8", "int8"),
+        ("Int16", "int16"),
+        ("Int32", "int32"),
+        ("Int64", "int64"),
+        ("Float32", "float32"),
+        ("Float64", "float64"),
+    ],
+)
+def test_all_type_comparison_with_false_result(dtype1, dtype2):
```

Comment:
please call dtype and expected

Reply:
after changing dtype1 to dtype and dtype2 to expected, do i need to change anything else?

Reply:
@phofl I made the change you wanted, is there anything else to merge this pr?